app/routes.py

```diff
@@ -13,3 +14,10 @@
 api.add_resource(AccountTypeResourceId, '/accounttype/<int:account_type_id>')
 api.add_resource(BranchDetailsResource, '/branchdetails')
 api.add_resource(BranchDetailsResourceId, '/branchdetails/<int:branch_details_id>')
+api.add_resource(AccountTransactionDetailsResource, '/accounttransactiondetails')
+api.add_resource(AccountTransactionDetailsResourceId, '/accounttransactiondetails/<int:account_transaction_details_id>')
+api.add_resource(TransactionTypeResource, '/transactiontype')
+api.add_resource(TransactionTypeResourceId, '/transactiontype/<int:transaction_type_id>')
+api.add_resource(FundTransferResource, '/fundtransfer')
+api.add_resource(FundTransferResourceId, '/fundtransfer/<int:fund_transfer_id>')
+api.add_resource(MiniStatementResources, '/ministatement')
```

Comment:
Mini statement should be for some account.

Reply:
Made the changes as suggested.